olive/model/hf_utils.py

```diff
@@ -32,7 +32,7 @@ def task_or_model_class_required(cls, v, values):
         if values["model_name"]:
             if not v and not values.get("task", None):
                 raise ValueError("Either task or model_class must be specified")
-            return v
+        return v
```

Comment:
Fix: when model_name is none, model_calss or task can not be set.

Based on following logics, model_name can be None.
![image](https://github.com/microsoft/Olive/assets/13343117/1ae6d9a7-61fa-4d8c-8105-8d612678e3d4)